crates/driver/openapi.yml

```diff
@@ -394,6 +394,10 @@ components:
           description: Id of the solution that should be executed.
           type: string
           example: "123"
+        blockDeadline:
+          description: The last block number in which the solution TX can be included.
+          type: string
+          example: "123"
```

Comment:
This can be a number.

Reply:
@MartinquaXD I agree, it was a number but I changed it to a `String`, because `solutionId` could be a number but it is sent a `String` , so I thought "there has to be a good reason for it", plus, to keep consistency. What is the reason `solutionId` is parsed as a `String` even tho it is an `u64`? should I keep the consistency?

Reply:
`SolutionId` is just some sort of identifier. Currently it's always a number but there is nothing preventing us to turn it into a string if that seems more helpful. Making it a string from the start means if we ever choose to change the underlying format no other code would have to be changed.
The block number on the other hand is and will always be a number so there is no reason to be extra flexible there.

Reply:
Why was this resolved? It's still a sting in the openapi spec.

Reply:
Ups, overwritten while fighting with my IDE for its automatic changes, fixed, good catch!